traefikee/values-ha.yaml

```diff
@@ -0,0 +1,63 @@
+# Custom values for High-Availability with Traefik Enterprise
+
+# This HA setup is without plugins.
+registry:
+  enabled: false
+
+controller:
+  replicas: 3
+  resources:
+    requests:
```

Comment:
You might add a comment saying that's it's need to be tuned according to infra/needs.

Reply:
Comment added.